src/docs/integrations/pagerduty.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "PagerDuty Integration"
+---
+
+## Create a PagerDuty app
+1. In PagerDuty, in the top right, click on "Developer mode"
+1. Select "Create New App"
```

Comment:
Can you make items like this bold? I like how the Slack integration docs are with this: https://develop.sentry.dev/integrations/slack/